go.mod

```diff
@@ -14,29 +14,28 @@ require (
 	github.com/lib/pq v1.10.6
 	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a
 	github.com/nxadm/tail v1.4.11
-	github.com/onsi/gomega v1.30.0
-	github.com/open-policy-agent/cert-controller v0.10.1
-	github.com/prometheus/client_golang v1.18.0
+	github.com/onsi/gomega v1.34.1
+	github.com/open-policy-agent/cert-controller v0.11.0
+	github.com/prometheus/client_golang v1.19.1
 	github.com/shirou/gopsutil/v3 v3.22.5
 	github.com/spf13/viper v1.9.0
 	github.com/tidwall/gjson v1.14.1
 	go.uber.org/mock v0.4.0
 	google.golang.org/grpc v1.58.3
-	google.golang.org/protobuf v1.33.0
-	k8s.io/api v0.29.3
-	k8s.io/apimachinery v0.29.3
-	k8s.io/client-go v0.29.3
-	k8s.io/code-generator v0.29.3
-	k8s.io/klog v1.0.0
-	k8s.io/kube-openapi v0.0.0-20231010175941-2dd684a91f00
-	sigs.k8s.io/controller-runtime v0.17.3
+	google.golang.org/protobuf v1.34.1
+	k8s.io/api v0.30.7
+	k8s.io/apimachinery v0.30.7
+	k8s.io/client-go v0.30.7
+	k8s.io/code-generator v0.30.7
+	k8s.io/klog/v2 v2.130.1
```

Comment:
Reverting klog version, then create dedicated issue for the klog version updating.